producer/disabled/disabled_producer_test.go

```diff
@@ -14,19 +14,24 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package producer_test
+package disabled
 
 import (
+	"encoding/json"
+	"github.com/RedHatInsights/insights-operator-utils/tests/helpers"
 	"testing"
 
-	"github.com/RedHatInsights/ccx-notification-service/producer"
 	"github.com/RedHatInsights/ccx-notification-service/types"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestDisabledProducer(t *testing.T) {
-	p := producer.DisabledProducer{}
-	_, _, err := p.ProduceMessage(types.NotificationMessage{})
+	p := Producer{}
+
+	msgBytes, err := json.Marshal(types.NotificationMessage{})
```

Comment:
it can be any type of JSON, right? not necessary of type `types.NotificationMessage{}`

Reply:
my point is that this can be a bit missleading maybe?

Reply:
In one of the future PRs I might add some checks inside ProduceMessage to verify that the content of `[]bytes` is indeed `types.NotificationMessage{}` :)

Reply:
will servicelog use `types.NotificationMessage{}` too? in that case, LGTM :)

Reply:
No, servicelog will actually use different type - I meant to check `types.NotificationMessage`  inside `KafkaProducer`. The fact that `ServiceLogProducer` accepts different data to produce was the reason to replace the type in the first place :)

Reply:
but the disable producer can accept any kind of data, right? why not use something like
```go
msgBytes, err := json.Marshal([]bytes())
```
?

Reply:
Oh, I get what you mean now! Ok, I will change it for disable producer then :)